client/windows/CHeroOverview.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * CHeroOverview.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+#include "../windows/CWindowObject.h"
+
+class CLabel;
+class CMultiLineLabel;
+class CFilledTexture;
+class CAnimImage;
+class CComponentBox;
+class CTextBox;
+
+class CHeroOverview : public CWindowObject
+{
+	const HeroTypeID & hero;
+	int heroIndex;
```

Comment:
hero & heroIndex are the same thing in the end - number.

Prefer to use HeroTypeID instead of int where possible

Reply:
Yes, but `heroIndex` is a little bit shorter than` hero.getNum()`.